packages/smooth_app/lib/pages/image/product_image_gallery_other_view.dart

```diff
@@ -146,4 +150,12 @@ class _RawGridGallery extends StatelessWidget {
       ),
     );
   }
+
+  ImageSize? _computeImageSize(double squareSize) => <ImageSize>[
+        ImageSize.THUMB,
+        ImageSize.SMALL,
```

Comment:
```suggestion
```

Reply:
It's OK with `THUMB` and `SMALL`, I don't really understand why you want to ditch them.

Reply:
I want to ditch them because I have experience in images and I remember that not all values are always populated, only 400 and FULL.
Just as an example, I've tried the latest barcode I used (https://fr.openfoodfacts.org/api/v3/product/3661344961326?fields=images):
```json
{
  "code": "3661344961326",
  "errors": [],
  "product": {
    "images": {
      "1": {
        "sizes": {
          "100": {
            "h": 100,
            "w": 75
          },
          "400": {
            "h": 400,
            "w": 300
          },
          "full": {
            "h": 3264,
            "w": 2448
          }
        },
        "uploaded_t": 1428607891,
        "uploader": "openfoodfacts-contributors"
      },
      "12": {
        "sizes": {
          "100": {
            "h": 100,
            "w": 99
          },
          "400": {
            "h": 400,
            "w": 396
          },
          "full": {
            "h": 2475,
            "w": 2448
          }
        },
        "uploaded_t": 1550253015,
        "uploader": "moon-rabbit"
      },
```
And so on.
It looks like "raw" images only have 100, 400 and full, never 200.

That's why I recommend 400 or FULL:
* 200 doesn't seem to exist for "raw" images, at least on a regular basis
* it's highly unlikely that user screens are less than 3*100 wide
* if we summon a 200 image that doesn't exist, that will look like unknown image
* the code would be much easier to maintain in a simple 400 or FULL choice
* the added value of the current code is "optimized image versus full image" (regardless of 100, 200 or 400) - and the added value of 100 versus 200 or 400 is probably less important